data/dataset-curation.md

```diff
@@ -249,7 +261,63 @@ If you choose to also fill in BIDS's optional [CHANGES](https://bids-specificati
 
 ## Derivatives Structure
 
-The [`derivatives`](https://bids-specification.readthedocs.io/en/stable/05-derivatives/01-introduction.html) are files generated from the top-level dataset such as segmentations or labels.
+This is a folder at the root of the dataset, which includes derivatives files generated from the top-level dataset such as segmentations or labeling.
+According to BIDS, these data should go under [`derivatives/`](https://bids-specification.readthedocs.io/en/stable/05-derivatives/01-introduction.html) folder, and follow the same folder logic as the `sub-*` data. 
+Example:
+
+```
+...
+...
+├── sub-XXX
+│   └── anat
+│       └──sub-XXX_T1w.nii.gz
+...
+...
+└── derivatives
+    ├── dataset_description.json
+    └── manual_labels
+        ├── sub-XXX
+        │   ├── anat
+        │   │   ├──sub-XXX_T1w_label-SC_seg.nii.gz
+        │   │   ├──sub-XXX_T1w_label-SC_propseg.nii.gz
+        │   │   ├──sub-XXX_T1w_label-SC_mask.nii.gz
+        │   │   ├──sub-XXX_T1w_label-GM_seg.nii.gz
+        │   │   ├──sub-XXX_T1w_label-WM_seg.nii.gz
+        │   │   ├──sub-XXX_T1w_label-centerline.nii.gz
+        │   │   ├──sub-XXX_T1w_label-disc.nii.gz
```

Comment:
maybe add an explanation for each item. Example:
```
        │   │   ├──sub-XXX_T1w_label-disc.nii.gz   <-- voxels located at the posterior tip of each disc, with values corresponding to SCT convention: https://spinalcordtoolbox.com/user_section/tutorials/registration-to-template/vertebral-labeling/labeling-conventions.html
```

Reply:
Regarding the disc labels, maybe we should specify that it includes all visible discs? I remember in spine generic, it was confusing if it was just 2 labels used for registration to template or all labels: https://github.com/spine-generic/data-multi-subject/issues/95

Reply:
> Regarding the disc labels, maybe we should specify that it includes all visible discs? I remember in spine generic, it was confusing if it was just 2 labels used for registration to template or all labels: [spine-generic/data-multi-subject#95](https://github.com/spine-generic/data-multi-subject/issues/95)

Good point! I explicitly stated "each intervertebral disc" in https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/94/commits/ee69cd16a587d684f5b4c86efa8c5d7c31980d2f